_content/essays/jhanas-and-the-lay-disciple_bodhi.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "The Jhānas and the Lay Disciples: According to the Pāli Suttas"
+authors:
+  - bodhi
+external_url: "http://dhammatalks.net/Books10/Bhikkhu_Bodhi_The_Jhanas_and_the_Lay_Disciples.htm"
+source_url: ""
+drive_links:
+  - "https://drive.google.com/file/d/1fKmtexkIEFaJJHZ64TUyka2VGMmthPKJ/view?usp=sharing"
+course: samadhi
+tags:
+  - theravada
```

Comment:
Hmm, does this essay conflict strongly with Mahayana doctrine?

Also: should we add `lay`?

Reply:
Bodhi mentions in the beginning that the jhana debate of recent decades is a Theravada one, so I thought it would be good to tag it. Happy to take it out if you'd like

Reply:
Ah! Replace with `jhana-controversy` then. I have a specific tag for that  😄